BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -25,6 +25,12 @@
 #import "BOXURLSessionManager.h"
 
 // Default API URLs
+/*
+ NOTE:  Currently when unversioned URLs are derived from the base URL,
+        it removes the last component of the URL path if and only if
+        the component begins with a number.
+        (e.g. "https://api.box.com/2.0" shortens to "https://api.box.com")
```

Comment:
For clarity (I always assume future-me is dumber than now-me), maybe we can add something like "If we change the base URL to not end with the API version, the change the API version to begin with a non-numerical character - the 'base URL without version' implementation will need to be updated as well." ?
I'm sure you can word that better, but something like that?